Cargo.toml

```diff
@@ -17,13 +17,13 @@ exclude = ["/.*"]
 [features]
 default = ["std"]
 alloc = []
-std = ["alloc", "instant"]
+std = ["alloc"]
+js = ["std", "getrandom"]
 
 [target.'cfg(all(target_arch = "wasm32", not(target_os = "wasi")))'.dependencies]
-instant = { version = "0.1", optional = true }
+getrandom = { version = "0.2", features = ["js"], optional = true }
```

Comment:
This cfg virtually means wasm32-unknown-unknown and wasm32-unknown-emscripten, but the getrandom js feature is for wasm{32,64}-unknown-unknown and is not used for emscripten.
https://github.com/rust-random/getrandom/blob/5aa4105c23065ca7477d1332022a017d6488a6c4/src/lib.rs#L269-L289